webpack/ForemanColumnExtensions/index.js

```diff
@@ -7,13 +7,64 @@ import {
   PackageIcon,
 } from '@patternfly/react-icons';
 import { Link } from 'react-router-dom';
-import { Flex, FlexItem, Popover, Badge } from '@patternfly/react-core';
+import {
+  Flex,
+  FlexItem,
+  Popover,
+  Badge,
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListDescription as Dd,
+  DescriptionListTerm as Dt,
+} from '@patternfly/react-core';
 import { translate as __ } from 'foremanReact/common/I18n';
 import RelativeDateTime from 'foremanReact/components/common/dates/RelativeDateTime';
 import { ContentViewEnvironmentDisplay } from '../components/extensions/HostDetails/Cards/ContentViewDetailsCard/ContentViewDetailsCard';
 import { truncate } from '../utils/helpers';
+import RepoIcon from '../scenes/ContentViews/Details/Repositories/RepoIcon';
 
 const hostsIndexColumnExtensions = [
+  {
+    columnName: 'bootc_booted_image',
+    title: <RepoIcon type="docker" customTooltip={__('Image mode / package mode')} />,
+    wrapper: (hostDetails) => {
+      const imageMode = hostDetails?.content_facet_attributes?.bootc_booted_image;
+      const digest = hostDetails?.content_facet_attributes?.bootc_booted_digest;
+      return (
+        <Flex>
+          {imageMode ?
+            <Popover
+              id="image-mode-tooltip"
+              className="image-mode-tooltip"
+              maxWidth="74rem"
+              headerContent={hostDetails.display_name}
+              bodyContent={
+                <Flex direction={{ default: 'column' }}>
+                  <DescriptionList isHorizontal>
+                    <DescriptionListGroup>
+                      <Dt>{__('Booted image')}</Dt>
```

Comment:
Do we want to be consistent with Insights which calls it [Running image](https://www.redhat.com/rhdc/managed-files/system%20details%20page%20in%20Red%20Hat%20Insights_image4.png)?

Reply:
Good idea, I can do that.

Should I also add the other info we collect? I can also show available, staged, and rollback so it's just like insights.

cc @ianballou

Reply:
I could also add any of those as options for additional columns you can add. Maybe just "Running image" would make sense. Thoughts?

Reply:
All these image information will be displayed in the image information card on details tab for each bootc hosts.

Reply:
Check out what Insights has for the pop up:
![image](https://github.com/user-attachments/assets/5c59b944-deb2-46f6-8d36-87f5b7483e44)

Instead of showing any data, they have a blurb about what a package mode machine is vs an image mode machine.

I prefer seeing the image information over a blurb about what image mode means. We might want to catch up with Maria on it.

Reply:
I didn't realize that the hover over text could be different from the window that pops up when you click on the logo. I suppose the blurb could go in the hover over part.

Reply:
I didn't add a blurb in the popover. The one in the screenshot is too downstream-oriented and I'm not sure what it would say for upstream. For now it just has the running image and digest; see what you think..